exchanges/kraken/kraken_wrapper.go

```diff
@@ -628,12 +629,40 @@ func (k *Kraken) CancelOrder(o *order.Cancel) error {
 	return err
 }
 
+// CancelBatchOrders cancels an orders by their corresponding ID numbers
+func (k *Kraken) CancelBatchOrders(orders []*order.Cancel) (order.CancelBatchResponse, error) {
+	var ordersList []string
+	for i := range orders {
+		if err := orders[i].Validate(orders[i].StandardCancel()); err != nil {
+			return order.CancelBatchResponse{}, err
+		}
+		ordersList = append(ordersList, orders[i].ID)
+	}
+
+	if k.Websocket.CanUseAuthenticatedWebsocketForWrapper() {
+		err := k.wsCancelOrders(ordersList)
+		return order.CancelBatchResponse{}, err
+	}
+
+	return order.CancelBatchResponse{}, common.ErrNotYetImplemented
```

Comment:
since there is no batch function for rest, can u just add common.ErrFunctionNotSupported

Reply:
done